templates/base.html.twig

```diff
@@ -1,19 +1,33 @@
+{# This is the base that all other views will extend #}
+
 <!DOCTYPE html>
 <html>
 <head>
     <meta charset="UTF-8">
     <link rel="icon" type="image/png" href="{{ asset('build/images/favicon.png') }}" />
```

Comment:
Is you favicon really build ?
If not, it would probably better to store it in a public/images directory